ts/Extensions/Annotations/NavigationBindings.ts

```diff
@@ -1486,6 +1487,51 @@ setOptions({
         }
     }
 });
+addEvent(H.Chart, 'render', function (): void {
+    var chart = this,
+        navigationBindings = chart.navigationBindings,
+        disabledClassName = 'highcharts-disabled-btn';
+
+    if (chart && navigationBindings) {
+        // check if the buttons should be enabled/disabled based on
```

Comment:
When the comment occupies the whole line please treat it as a sentence- start with a capital letter, ends with a dot.
`// My comment sentence.`

Comments at the end of the code could be treated as words.
` if (chart.stockTools && chart.stockTools.toolbar) { // some comment`